src/collections/events/2024/hacktoberfest-2024/index.mdx

```diff
@@ -0,0 +1,42 @@
+---
+title: "Hacktoberfest Prep 2024: Extending Meshery Models"
+description: "Prepare for Hacktoberfest with Layer5 MeshMates and Maintainers. Join seasoned open source maintainers, hack through easy issues, and grow your skills. Discover Newcomers Resources and explore the Layer5 Community Handbook."
+thumbnail: ./hacktoberfest2024.jpg
+darkthumbnail: ./hacktoberfest2024.jpg
+date: 2024-09-27
+eurl: /community/calendar
+type: Event
+speakers: ["Vivek Vishal", "Sudhanshu Dasgupta", "Uzair Shaikh", "Shlok Mishra", "Hargun Kaur", "Aabid Sofi", "Awani Alero"]
+published: true
+upcoming: true
+register: true
+---
+
+import { Link } from "gatsby" ;
+
+import hacktoberfestBadge from "./hacktoberfest-contributor.png";
+
+<p>
+<Link to="/programs/hacktoberfest">Hacktoberfest</Link> is just around the corner. You'd be surprised to know how small changes can make a big impact on a project. The more a project scales, the easier it is for small errors to live within it. You can make a real difference by fixing these relatively easier issues and continue to grow from there.
+</p>
+
+<p>
+Prepare for Hacktoberfest and ready yourself for contributing to CNCF projects the easy way with Layer5 <Link to="/community/meshmates">MeshMates</Link> and Maintainers. Spend an hour with seasoned, open source maintainers and hack your way through. To get a head start, see our <Link to="/community/newcomers">Newcomers Resources</Link>. Get familiar with the Layer5 community through our <Link to="/community/handbook">Community Handbook</Link>.
+</p>
+
+<ul>
+    <li><strong>DATE:</strong> September 27, 2024</li>
+    <li><strong>TIME:</strong> 10:00 AM CT | 8:30 PM IST</li>
+    <li><strong>MEET:</strong> <a href="https://meet.layer5.io/community"> Zoom</a></li>
+</ul>
+
+<p> Resources:
+    <ul>
+    <li><Link to="https://github.com/issues?q=is%3Aopen+is%3Aissue+archived%3Afalse+org%3Alayer5io+org%3Ameshery+org%3Aservice-mesh-performance+org%3Aservice-mesh-patterns+label%3A%22hacktoberfest%22+">Issues for Hacktoberfest</Link></li>
```

Comment:
Make sure this has layer5labs as an org, too. Update the place where you got this link, too.